### DISCUSSION TITLE
Roll back Proxmox VE 7 Post Install

### DISCUSSION THREAD

**Discussion by Jens-Wymeersch** [tteck/Proxmox#487] [Help Needed] [answered]
Dear @tteck,

Thank you for all the work you have done. I've tested a lot of it during the last few months. Based on the different projects, I've decided to buy a licence from proxmox. In order to cleanly activate this, I need to be able to roll back the installation of the post install. Do you have a script I would be able to run to do this ? 

Thank you so much,
Jens

**Comment by tteck** [comment, marked as answer]
You can make all changes in the _Repositories_ tab
Disable the repositories in red and enable the Enterprise repo
 
![Screenshot 2022-09-06 6 10 23 AM](https://user-images.githubusercontent.com/21342670/188609599-efefa6f6-daa7-4361-97d4-c0e0c7e63b08.png)

**Reply by tteck** [reply]
Also, to undo the no-nag
```
rm /etc/apt/apt.conf.d/no-nag-script
apt --reinstall install proxmox-widget-toolkit
```
clear all browser cache

**Comment by Jens-Wymeersch** [comment]
@tteck I've just done the commands indicated in your last post. However, I'm still seeing the following 

<img width="1076" alt="image" src="https://user-images.githubusercontent.com/70786498/188613286-89a30700-8412-4919-b22a-ea197011eebd.png">


although that, I have a valid licence

**Reply by tteck** [reply]
I know nothing about activating a subscription, as I've never done that. 🤷🏻‍♂️

**Reply by tteck** [reply]
May need to reboot Proxmox.

**Reply by tteck** [reply]
Did you select the tab “Subscription” – then click “Upload Subscription Key”, 
and insert your key?

**Reply by Jens-Wymeersch** [reply]
Yes, I did this and still get the annoying message and the error message 

in the system report there, I get the following :

```
# pvesubscription get
checktime: 1662461389
key: mykey
level: c
nextduedate: 2023-09-03
productname: Proxmox VE Community Subscription 1 CPU/year
```

Although that, it is not reflected in the rest of the console. I don't know what to do to get this sorted and the proxmox folks basically said, you installed stuff, it is your responsibility.

**Reply by tteck** [reply]
Wow, that's the support you get for a paid subscription? I'll pass.

You can always backup everything to a USB drive and re-install Proxmox then restore your backups, but the post install script only sets repositories and stops the nag.

**Reply by tteck** [reply]
Also, did you reboot Proxmox?

**Reply by Jens-Wymeersch** [reply]
Yes I rebooted. I'm scratching my hair, what it can be. I'm running v7 now for 6 months. I remember a friend of mine helped me initially with the setup. What he did, I don't really know ?
 Is there some kind of way to figure out this out ?

**Reply by tteck** [reply]
One last thing to try, clear all browser cache

**Reply by Jens-Wymeersch** [reply]
And that did the trick... We have success !
